EIPS/eip-00.md

```diff
@@ -0,0 +1,790 @@
+
+If an AA transaction is included in a block, returns the following values in addition to the existing fields:
+
+| Name                       | Value                                                                        |
+|----------------------------|------------------------------------------------------------------------------|
+| sender                     | Address of the sender of this transaction                                    |
+| nonce                      | The transaction nonce value                                                  |
+| paymaster                  | Address of the Paymaster if it is paying for the transaction, null otherwise |
+| deployer                   | Address of the Deployer if it is included in the transaction, null otherwise    |
+| senderCreationGasUsed      | The amount of gas actually used by the sender deployment frame               |
+| senderValidationGasUsed    | The amount of gas actually used by the sender validation frame               |
+| paymasterValidationGasUsed | The amount of gas actually used by the paymaster validation frame            |
+
+Accepts hash of Transaction Type `AA_TX_TYPE`.
+
+If an AA transaction is included in a block, returns the following values in addition to the existing fields:
+
+| Name                            | Value                                                                                          |
+|---------------------------------|------------------------------------------------------------------------------------------------|
+| status                          | Either 1 (success) or 0 (failure) status of the execution frame                                |
+| executionGasUsed                | The amount of gas actually used by the execution frame                                         |
+| postPaymasterTransactionStatus  | Either 1 (success), 0 (failure), or `null` (did not run) status of the `postPaymasterTransaction` frame |
+| postPaymasterTransactionGasUsed | The amount of gas actually used by the paymaster `postPaymasterTransaction` frame                       |
+
+Note that the field `to` is not included as there is no clear `target` in an `AA_TX_TYPE` transaction.
+
+#### `eth_call`
+
+Accepts Transaction Type `AA_TX_TYPE` with all fields except `from` and `callData` optional.
+
+Returns the return value of [the `sender` execution frame](#sender-execution-frame).
+
+If provided with `paymasterData` and `deployerData` also executes the corresponding frame.
+
+If any of the frames reverts the call returns the revert data of each reverted frame.
+
+#### `eth_estimateGasAccountAbstraction`
+
+Accepts Transaction Type `AA_TX_TYPE` with fields `validationGasLimit`, `paymasterGasLimit`, `callGasLimit` optional.
+
+Returns `{validationGasLimit, paymasterGasLimit, callGasLimit, builderFee}` object.
+
+Note that the `deployerData` and `paymasterData` fields are required for a consistent result.
```

Comment:
Might make sense to define what a "dummy signature" is, and its requirements here.

Reply:
We can't define its requirements, as they are set by the account itself.  But we should mention that the account needs to support one, in order to support gas estimation.